go/parquet/file/file_reader_test.go

```diff
@@ -19,6 +19,9 @@ package file_test
 import (
 	"bytes"
 	"encoding/binary"
+	"github.com/apache/arrow/go/v12/parquet"
+	"github.com/apache/arrow/go/v12/parquet/schema"
+	"github.com/stretchr/testify/require"
```

Comment:
can you run `go fmt` on this / `goimports` so it reorders the imports sorting them appropriately? that's likely what's causing the failing dev script CI tests, these are getting ordered differently resulting in the diffs not being what is expected.

Reply:
I should really add a workflow for the bot to auto-run the formatting like we have for C++ and clang-format...

Reply:
Done! Thanks for the pointer.

> I should really add a workflow for the bot to auto-run the formatting like we have for C++ and clang-format...

I can take a look if you guys need help on the go front :-)

Reply:
Hmm, still have failing checks, I ran:
`go fmt -d file_reader.go file_reader_test.go` and see no diffs 🤔

Reply:
you needed to remove the empty line and then use goimports to sort the imports. I've done so and pushed the fix.